### ISSUE TITLE
Model fine-tuning takes a large amount of steps

### ISSUE THREAD

**Issue by uchendui** [uchendui/maml#1]
Fine tuning the network after meta training is not producing results similar to the paper. The problem could lie in how gradients are passed through the adapted parameters in the "create_model" function. 

EDIT: After further experiments, fine-tuning is able to fit the curve quite well after a large number of training steps. This differs from the paper where the sine curves are fit after ~10 steps during fine-tuning.
![maml_finetune](https://user-images.githubusercontent.com/14854496/63442044-67153d00-c400-11e9-8944-40283dccb6e3.png)
